docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
```

Comment:
Would `Sub-Resolutions` not be more accurate as the section heading.  "Pyramid" is fine, but it's not as descriptive.  `OME-TIFF` is redundant.